autopush/web/registration.py

```diff
@@ -68,8 +68,11 @@ def convert_chid(self, data):
 
 
 class TokenSchema(SubInfoSchema):
-    """Validates that token and optionally key/channelID"""
+    """Filters allowed values from body data"""
     token = fields.Str(allow_none=True)
+    # TODO: Temporarily allow 'aps' definition data for iOS.
```

Comment:
Ok, how about removing the comment entirely? Better to say nothing than mislead, no?